### PR TITLE
FIFO pop

### PR DESCRIPTION
**Pull request by steinkel**
* Add option to retrieve jobs FIFO
* Fix docker build

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,7 +27,6 @@ before_script:
   - sudo apt-get update
   # beanstalkd
   - sudo apt-get install -qq --force-yes beanstalkd
-  - sudo beanstalkd -d -l 127.0.0.1 -p 11300
```
Comment (josegonzalez) [inline .travis.yml]: Why this change

Comment (steinkel) [reply .travis.yml]: looks like the daemon is now started on apt-get install, port was reported as used in that line
